contracts/commodity/BasicCommodity.sol

```diff
@@ -521,9 +562,9 @@ contract BasicCommodity is UnstructuredOwnable, EIP820Implementer, ICommodity {
 
   //TODO (jaycen) PRELAUNCH fix/remove this (if we need it for compatibility reasons  -- disabling for now)
   /** @notice Authorize a third party '_operator' to manage (send) 'msg.sender''s tokens. */
-  function authorizeOperator(address) public pure {
-    revert();
-  }
+  // function authorizeOperator(address) public pure {
+  //   revert();
+  // }
```

Comment:
No, leaving it for now, I need to revisit the official standard as 721 has since been officially "accepted" and we started this long before that was done. Since 721s are now visible in etherscan id like to make sure we have all requisite function definitions pre launch